src/TipBox.vue

```diff
@@ -74,11 +74,20 @@
         type: Boolean,
         default: false,
       },
+      noIcon: {
+        type: Boolean,
+        default: false,
+      },
+      noBackground: {
+        type: Boolean,
+        default: false,
+      },
+      noBorder: {
+        type: Boolean,
+        default: false,
+      }
     },
     computed: {
-      isDefault() {
-        return this.type === 'none'
-      },
```

Comment:
Why is `none` being removed?

Reply:
I'm not removing the `none` type, `none` is the type given to default style boxes.

The check on whether or not to render the `icon-wrapper` is not longer based on the type of the box, whether it is a default type or not. Because a default type box can still have an icon.

Example:
```
<box icon=":rocket:">
    Lorem ipsum dolor sit amet,
</box>
```
![image](https://user-images.githubusercontent.com/27397021/74598277-cc917000-50a9-11ea-8ee7-1e6c28d581d7.png)